src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,23 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    public HibernateUtil() {
+    }
```

Comment:
To prevent creating HibernateUtil objects, add a private default constructor. This aligns with the checklist item: 'Add a private default constructor to the HibernateUtil class in order to prevent creating HibernateUtil objects.'